docs/VLCB4in4out_README.md

```diff
@@ -83,21 +84,21 @@ OP_CODE | HEX | Function
  OPC_ASON | 0x98 | Short event on
  OPC_ASOF | 0x99 | Short event off
 
-## Set Up
+## Events
 
-The module is initialised by pressing the VLCB button (formerly CBUS button) for
-6 seconds when the green LED goes off and the Yellow LED flashes. A module name and
-Node Number can then be set via the FCU in the normal manner.
+### Event Variables
 
-## Events
+Event Variable 1 (EV1) identifies a consumed only event if its value is zero.
```

Comment:
Is this sentence correct. Isn't consumed events controlled by EV2 and upwards?

Reply:
That's true but the sentence is correct in that if EV1 is zero then the event is consumed only.  If it is non-zero, then it is a produced event and possibly consumed as well if EV2 onwards are non-zero.